lib/menu.rb

```diff
@@ -0,0 +1,25 @@
+class Menu
+
+  attr_reader :dishes
+  
+  DEFAULT_MENU = {
+      spring_rolls: 5,
+      wonton: 10,
+      congee: 10,
+      chow_mein: 15,
+      hotpot: 20
+    }
+
+  def initialize(dishes = DEFAULT_MENU)
+    @dishes = dishes
+  end
+
+  def print_menu
+    @dishes.map do |name, price|
+      "#{name.to_s.split("_").map(&:capitalize).join(" ")}: £#{price}"
```

Comment:
Maybe split this bit of code up e.g. formatted_name = name.to_s.split("_") then do `#{formatted_name.map....)